spec/lib/data_update_scripts/add_single_resource_admin_role_to_users_with_tech_admin_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20210203104631_add_single_resource_admin_role_to_users_with_tech_admin.rb",
+)
+
+describe DataUpdateScripts::AddSingleResourceAdminRoleToUsersWithTechAdmin do
+
+  before do
+    users = create_list(:user, 5)
+    User.first.add_role(:tech_admin)
```

Comment:
I think it'd be easier to create three distinct users:

```ruby
tech_admin1 = create(:user, :tech_admin)
tech_admin2 = create(:user, :tech_admin)
admin = create(:user, :admin)
```

then you can assert against things like `tech_admin1.reload.roles`, etc. which seems better to me than going through the AR query interface.

Reply:
Done ✅